lib/tasks/collection_type_global_id.rake

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+namespace :hyrax do
+  namespace :collections do
+    desc 'Update CollectionType global id references for Hyrax 3.0.0'
+    # Note: the definition of collection_type_gid= changed in Hyrax 5.0. This
+    # rake task is known to work for updates to versions prior to Hyrax 5.
+    # Use for later versions is unknown. 
+    task update_collection_type_global_ids: :environment do
+      # Account.find_each do |account|
+        next if account.search_only.eql? true
+        next if account.is_public.eql? false
```

Comment:
Why are we ignoring private tenants?

Reply:
I will remove that. I added that code when I verified that all collections got updated in the public tenants, because running the code the first time crashed my rails console. I wanted to run a smaller subset the second time over, just to ensure that it looked like everything was corrected. But you're right that in general we'd want to run for all tenants.